views/guide.lavender

```diff
@@ -91,7 +91,7 @@ block content
             | Add the lavender submodule with 
             code git submodule add git@github.com:golavender/lavender.git (folder to 
             | clone to) and include it with 
-            code require "(where you put lavender)/src/Lavender/lavender.php   
+            code require '(where you put lavender)/src/Lavender/lavender.php   
```

Comment:
Think we need a closing ' at the end of this line.

code '(where you put lavender)/src/Lavender/lavender.php'